apps/gateway/test/champions_test.exs

```diff
@@ -301,6 +302,13 @@ defmodule Gateway.Test.Champions do
 
       assert Enum.count(campaigns.campaigns) > 0
 
+      # Check the campaign numbers match the expected range
+      campaign_numbers = 1..length(campaigns.campaigns)
+
+      Enum.each(campaign_numbers, fn number ->
+        assert Enum.any?(campaigns.campaigns, &(&1.campaign_number == number))
+      end)
```

Comment:
What are we testing here? that the list of campaigns has the correct numbers? if this is due to the previous comment it wasn't my intention, the aim of the comment was to check that the get_campaigns returns all the campaigns available in the database